to-do-ui/src/app/components/task/task.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="card-container">
+	<p-card (mouseenter)="showButtons = true" (mouseleave)="showButtons = false"
+		[ngStyle]="{'background-color': task.isCompleted ? 'greed' : null}">
+		<div class="card-header-container">
+			<h3>{{task.title}}</h3>
+			<div class="container" *ngIf="this.showButtons">
```

Comment:
```suggestion
			<div *ngIf="showButtons" class="container">
```
ngIf first, because it is clearer and you can get rid of "this." in html